views/01UCS_LAL-PRIMO_REDESIGN/css/ucla/account.css

```diff
@@ -175,3 +181,23 @@ prm-fines
   line-height: 160%;
   letter-spacing: 0.2px;
 }
+
+/* Account > Fines + Fees: Allow long titles to wrap */
+prm-fines md-list md-list-item h4 {
+  white-space: unset !important;
+}
+
+/* Account > Fines + Fees: Enforce 2-column display of data for all cards */
+div.flex-xs-100.flex {
+  min-width: unset;
+}
+
+/* Account -> Fines + Fees: U/Body/Button for link to payment page */
+.tab-content-header .md-button.button-as-link {
+  color: var(--color-black);
+  font-family: Proxima Nova;
```

Comment:
Changing this to `proxima-nova` fixes the font display for me.

Reply:
Good catch, I forgot that the font name in the Figma design is not what's needed in CSS.  Fixed!